### PR TITLE
docs(react): Add example for useVisibilityEvent.

### PR DESCRIPTION
**Pull request by rjsdnql123**
## Overview

I have added an example for the missing useVisibilityEvent.

## PR Checklist

- [x] I read and included theses actions below

1. I have read the [Contributing Guide](https://github.com/toss/slash/blob/main/.github/CONTRIBUTING.md)
2. I have written documents and tests, if needed.

### CODE REVIEW

File: packages/react/react/src/hooks/useVisibilityEvent.en.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
 # useVisibilityEvent
 
 Call the callback function when the document's visibilitychange event occurs.
+
+## Example
+
+```tsx
+type VisibilityState = 'hidden' | 'visible';
```
Comment (HoseungJang) [inline packages/react/react/src/hooks/useVisibilityEvent.en.md:8]: There is [the built-in type `DocumentVisibilityState` in Typescript](https://github.com/microsoft/TypeScript/blob/cbda2fcce95d7ac82841cee631f07b73be07c0f0/src/lib/dom.generated.d.ts#L27834).

File: packages/react/react/src/hooks/useVisibilityEvent.en.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+
+function MyComponent() {
+  const visibleCallbackFunc = (visibilit: VisibilityState) => {
```
Comment (HoseungJang) [inline packages/react/react/src/hooks/useVisibilityEvent.en.md:11]: `visibilit` looks a typo.

Comment (rjsdnql123) [reply packages/react/react/src/hooks/useVisibilityEvent.en.md]: I made a mistake.. Thank you!!

**Comment by rjsdnql123** [comment]
@HoseungJang   Can you please double-check once again?!
Thank you : )

**Review by HoseungJang — APPROVED** [review]

**Pull request merged**